purchases/src/main/kotlin/com/revenuecat/purchases/Purchases.kt

```diff
@@ -1319,15 +1320,15 @@ class Purchases @VisibleForTesting(otherwise = VisibleForTesting.PRIVATE) intern
             )
 
             val billingWrapper = BillingWrapper(
-                BillingWrapper.ClientFactory((context.getApplication()).applicationContext),
-                Handler((context.getApplication()).mainLooper)
+                BillingWrapper.ClientFactory(application),
```

Comment:
As you said, this was redundant the applicationContext of an application is a reference to the application itself